styles.css

```diff
@@ -0,0 +1,38 @@
+@import "./Components/Avatar/avatarstyles.css";
```

Comment:
styles keyword can be removed from every component.
```suggestion
@import "./Components/Avatar/avatars.css";
```